pycqed/instrument_drivers/meta_instrument/qubit_objects/CCL_Transmon.py

```diff
@@ -3680,6 +3707,116 @@ def measure_gst_1Q(self,
         a = ma2.GST_SingleQubit_DataExtraction(label='Single_qubit_GST')
         return a
 
+    def make_qubit_dac_arc(self, dac_values=None, polycoeffs=None, MC=None,
```

Comment:
Besides the `MC` argument, you should also add the `nested_MC` argument as this experiment requires two measurement controls.